fofix/core/Theme.py

```diff
@@ -1933,9 +1931,9 @@ def renderAlbumArt(self, scene):
                     if abs(d) < 1.2:
                         label = scene.itemLabels[i]
                         if label == "Random":
-                            label = scene.img_random_label
+                            label = scene.img_random_label.texture
                         if not label:
-                            label = scene.img_empty_label
+                            label = scene.img_empty_label.texture
```

Comment:
With this line, the `empty_label.png` file will be required to navigate between songs. I think you should check if the file exists (if `scene.img_empty_label` is not `None`) because this could break some existing themes.

Reply:
Not my code, but OK, it's fixed.